simple_salesforce/api.py

```diff
@@ -2,6 +2,8 @@
 
 
 # has to be defined prior to login import
+import decimal
```

Comment:
I think I mistakenly added this import here in the patch. I think it should got down in the regular import area. `DEFAULT_API_VERSION` is what needs to be defiend prior.